tests/python/unittest/test_ndarray.py

```diff
@@ -819,7 +821,11 @@ def get_large_matrix():
 
         # test for ret_typ=indices
         nd_ret_topk = mx.nd.topk(a_nd, axis=1, ret_typ="indices", k=3, is_ascend=True).asnumpy()
-        assert nd_ret_topk.dtype == np.float32  # Test the default dtype
+        # Test the default dtype
+        if is_large_tensor_enabled:
```

Comment:
changed default type from float for int32/64 based on whether large tensor support is enabled or not